src/tablecloth/time/derived.clj

```diff
@@ -0,0 +1,25 @@
+(ns tablecloth.time.derived)
+
+(defn add-derived
```

Comment:
Wondering a bit about the naming here? I wonder if it could be a bit more specific, or rather does it need to be so general? Are we going to add other types of "derived" data?